core/modules/node/node.module

```diff
@@ -1863,6 +1865,10 @@ function node_block_info() {
     'description' => t('Displays a node in a block.'),
     'class' => 'NodeBlock',
   );
+  $blocks['empty'] = array(
+    'info' => t('Create content'),
+    'description' => t('A prompt to create some content with appropriate links.'),
+  );
```

Comment:
What was related to this issue was that we were removing the old "No front page content created yet" text and links to go create some. I suppose I thought this was killing two birds with one stone.